### PR TITLE
chore(button): improve Execute transaction button #870

### LINKED ISSUE #870: improve Execute transaction bottom
We need to fix the button, put some space between the words
image

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- Resolve #870 

### Steps to test

1. Go to https://mainnet.eosio.online/accounts and see contract tables `table` input form and execute transaction in` Contract Actions`

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: webapp/src/components/ContractActionForm/index.js [hunk 1]
```diff
@@ -261,7 +261,7 @@ const ContractActionForm = ({ accountName, action, abi, onSubmitAction }) => {
           color="primary"
           onClick={handleSubmit}
         >
-          {t('executeTransaction')}
+          {t('Execute Transaction')}
```
Comment (Torresmorah) [inline webapp/src/components/ContractActionForm/index.js:264]: I think you need to check the language files, to see if that entry exists.
Did you check that the text changes when you switch to Spanish?

**Review by Torresmorah — CHANGES_REQUESTED** [review]

File: webapp/src/components/ContractActionForm/index.js (hunk 1 above)
Comment (xavier506) [reply webapp/src/components/ContractActionForm/index.js:264]: good catch

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by tetogomez — CHANGES_REQUESTED** [review]
remove Box component and use `div` from this PR

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**